.travis.yml

```diff
@@ -23,11 +23,11 @@ script: tools/travis-test.sh -p $PRESET
 
 after_failure:
         - cat `ls -1 -d -t apps/ejabberd/logs/ct_run* | head -1`/apps.ejabberd.logs/run.*/suite.log
-        - cat dev/mongooseim_node1/etc/ejabberd.cfg
-        - if [ -s dev/mongooseim_node1/log/crash.log ]; then cat dev/mongooseim_node1/log/crash.log; fi
-        - if [ -s dev/mongooseim_node2/log/crash.log ]; then cat dev/mongooseim_node2/log/crash.log; fi
-        - tail -100 dev/mongooseim_node1/log/ejabberd.log
-        - tail -100 dev/mongooseim_node2/log/ejabberd.log
+        - cat _build/mim1/rel/mongooseim/etc/ejabberd.cfg
```

Comment:
I'm wondering if we still need this since most of this is archived on S3, f.e here: http://opensource.erlang-solutions.com/mongooseim-ct-reports/s3_reports.html?prefix=PR/1196/2768/internal_mnesia.18.3/logs/2017-02-24_10.04.13/